ebusd-0.5.x/ochsner/ote2.csv

```diff
@@ -0,0 +1,43 @@
+*r,ochsner,,,,15,0621,,,,,,,,,,,,,
+*w,ochsner,,,,15,0621,,,,,,,,,,,,,
+r,temperature,outside,Au�entemperatur,,,,00800040,,,parameter,,,,,,temperature,,,,
+r,temperature,water,Warmwasser Temperatur,,,,00840040,,,parameter,,,,,,temperature,,,,
+r,temperature,buffer,Vorlauftemperatur,,,,00e00040,,,parameter,,,,,,temperature,,,,
+r,heating,status,,,,,77800008,,,parameter,,,,status,,UCH,,,,
```

Comment:
the common prefix (e.g. heating and ID starting with 77) could be nicer written like this (unless the ID suffix 0008 is of any relevance):
*r,heating,,,,15,0621,77,,,IGN:2,,,,,,,,,,
r,,status,,,,,80,,,parameter,,,,status,,UCH,,,,
r,,outside,,,,,81,,,parameter,,,,,,temperature,,,,
...
*r,boiler,,,,15,0621,79,,,IGN:2,,,,,,,,,,
r,,status,,,,,80,,,parameter,,,,status,,UCH,,,,
...